pkg/reconciler/apiserversource/resources/receive_adapter_test.go

```diff
@@ -62,8 +63,8 @@ func TestMakeReceiveAdapter(t *testing.T) {
 	trueValue := true
 	want := &v1.Deployment{
 		ObjectMeta: metav1.ObjectMeta{
-			Namespace:    "source-namespace",
-			GenerateName: "apiserversource-source-name-",
```

Comment:
I would have prefer a `-` in-between the name and UID. No big deal.

Reply:
Done in #1626.